iothub_client/inc/internal/iothub_transport_ll_private.h

```diff
@@ -91,12 +92,13 @@ extern "C"
     typedef int(*pfIoTHubTransport_SetCallbackContext)(TRANSPORT_LL_HANDLE handle, void* ctx);
 
 #define TRANSPORT_PROVIDER_FIELDS                                                   \
-pfIotHubTransport_SendMessageDisposition IoTHubTransport_SendMessageDisposition;  \
+pfIotHubTransport_SendMessageDisposition IoTHubTransport_SendMessageDisposition;    \
 pfIoTHubTransport_Subscribe_DeviceMethod IoTHubTransport_Subscribe_DeviceMethod;    \
 pfIoTHubTransport_Unsubscribe_DeviceMethod IoTHubTransport_Unsubscribe_DeviceMethod;\
 pfIoTHubTransport_DeviceMethod_Response IoTHubTransport_DeviceMethod_Response;      \
 pfIoTHubTransport_Subscribe_DeviceTwin IoTHubTransport_Subscribe_DeviceTwin;        \
 pfIoTHubTransport_Unsubscribe_DeviceTwin IoTHubTransport_Unsubscribe_DeviceTwin;    \
+pfIoTHubTransport_GetDeviceTwinAsync IoTHubTransport_GetDeviceTwinAsync;            \
```

Comment:
>pfIoTHubTransport_GetDeviceTwinAsync IoTHubTransport_GetDeviceTwinAsync; [](start = 0, length = 72)

Why add this to the middle?  In this case if it's in the middle, and someone is using a DLL or SO and only replaces the transport library then it will crash as oppose to adding it to the end where it will be backwards compatible.

Reply:
got it. Makes sense.